IBPSA/Fluid/BaseClasses/MassFlowRateMultiplier.mo

```diff
@@ -0,0 +1,65 @@
+within IBPSA.Fluid.BaseClasses;
+model MassFlowRateMultiplier "Model that multiplies the mass flow rate"
+  extends IBPSA.Fluid.Interfaces.PartialTwoPort;
+
+  parameter Real k "Gain for mass flow rate";
+initial equation
+  assert(k > Modelica.Constants.small or -k < -Modelica.Constants.small,
```

Comment:
Is there a reason for putting this in an initial equation section? You can also add it in the equation section. In that case, I have the impression that dymola already evaluates the assert during translation when it only depends on parameters. In that way you get an error message earlier.

Reply:
I verified this and the check is only evaluated at translation when `Evaluate=true`, for both cases.